Apps/W1/HybridGP/app/src/Migration/Customers/GPCustomerAddress.Table.al

```diff
@@ -97,6 +100,14 @@ table 4048 "GP Customer Address"
             if (CopyStr(ShipToAddress."Fax No.", 1, 14) = '00000000000000') then
                 ShipToAddress."Fax No." := '';
 
+            if GPSY01200.Get('CUS', CUSTNMBR, ADRSCODE) then
+                EmailAddress := CopyStr(GPSY01200.INET1.Trim(), 1, MaxStrLen(ShipToAddress."E-Mail"));
+
+#pragma warning disable AA0139
+            if MailManagement.ValidateEmailAddressField(EmailAddress) then
```

Comment:
I have seen in customer data where they have multiple email addresses in one line. Are we to account for these or are we relying on the customer to fix the bad data before migrating? Do we have a check in the pre-migration tool?